srcalc/src/main/java/gov/va/med/srcalc/domain/model/DiscreteTerm.java

```diff
@@ -135,6 +135,6 @@ public int hashCode()
     public String toString()
     {
         return String.format(
-                "%s=%s?->%s", getVariable(), getOption(), getCoefficient());
+                "%s=%s->%s", getVariable(), getOption(), getCoefficient());
```

Comment:
The BooleanTerm had a comment to this effect but DiscreteTerm didn't. Its not really a boolean so I thought this was a cut-n-paste error.  I think its clearer w/o the ? but obviously it doesn't matter much either way.